privval/remote_signer.go

```diff
@@ -147,6 +147,12 @@ func (sc *RemoteSignerClient) Ping() error {
 	return nil
 }
 
+// SendPoisonPill allows us to ask the remote signer to shut down.
+func (sc *RemoteSignerClient) SendPoisonPill() error {
+	// we need no response when sending a poison pill message
+	return writeMsg(sc.conn, &PoisonPillMsg{})
+}
```

Comment:
I guess we can do away with all this `PoisonPill` related code now (as per https://github.com/tendermint/kms/pull/162)

Reply:
Definitely - it's now removed 👍